### ISSUE TITLE
iPad 10 Safari inconsistencies

### ISSUE THREAD

**Issue by kelvinballoo** [google/site-kit-wp#9776]
## Bug Description

When I check on iPad 10 safari, I see some inconsistencies.
Documented in the details section:

<Details> 
For the 'New visitors' tile, there is some overlapping text.
Also for the other tiles, I feel the content is very close to the title of each tile.

![Image](https://github.com/user-attachments/assets/3756cf7b-f111-4356-a154-b65ac4d69378)

Notice how the 3rd tile is bigger than the first 2, when it's the same information being displayed.

![Image](https://github.com/user-attachments/assets/17afe826-f878-414e-8946-510cb80a2a20)

</Details>

## Steps to reproduce

1. Set up KM and use the metrics as per the screenshots, eg:
  * top recent trending pages
  * most popular authors by pageviews
  * top categories by pageviews
  * most popular content by pageviews
  * most engaging pages
  * least engaging pages
  * top traffic source
  * most engaged traffic source

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* Metric tiles should not have title and content overlapped on any screen
* On smaller screens (tablet and bellow) there should be no extra space height-wise if the content of the metric items shown in the same row, is either the same content or content of equal count (i.e same or similar content should not cause dramatic differences in spacing/height)

## Implementation Brief

* [ ] In https://github.com/google/site-kit-wp/blob/39194b24f31cf96d0fbffd99a87bd2bc6b89eab9/assets/sass/components/key-metrics/_googlesitekit-km-widget-tile.scss#L275-L281
  * Remove the `grid-template-rows` override for tablet media query `$bp-tablet`, and keep the existing media query for `$bp-desktop`, but replace it to target from `$bp-tablet` instead

### Test Coverage

* Update any failing VRT

## QA Brief

* Follow the steps to reproduce
* Text should not appear overlapping like in the issue description/screenshot

## Changelog entry

* Fix iPad 10 Safari inconsistencies.

**Comment by 10upsimon** [comment]
@zutigrm AC LGTM ✅ , I did enhance the height sentence a little as this applies to conetnt of a similar length and not just the same content.

**Comment by eugene-manuilov** [comment]
IB ✔

**Comment by kelvinballoo** [comment]
### QA Update ⚠ 

This is a bit tricky.
I tested on a couple of devices, all are working fine, except for iPad 10 Safari 16.
I chose iPad 10 because that was the latest in browserstack but maybe the old version 16 of Safari is what makes it not work.
If I test on iPad 9 Safari 18, the issue doesn't appear.

Photos are attached. 

My gut feel is that this is an edge case on an old safari version and it might not be worth investigating for iPad 10 Safari 16. Unless there is a quick fix for it of course.

Let me know what you think.

<Details>
iPad 10 - Not tested good

<img width="916" alt="Image" src="https://github.com/user-attachments/assets/fe3f5cec-a27c-43a1-856b-2aac738d743b" />

Other devices - Tested good

<img width="910" alt="Image" src="https://github.com/user-attachments/assets/3e3653bc-5702-49bd-b048-25ed6029d012" />

<img width="1158" alt="Image" src="https://github.com/user-attachments/assets/e82b0362-d34a-4264-851e-51d7a65aa1e7" />

<img width="507" alt="Image" src="https://github.com/user-attachments/assets/54c3752d-2c08-42a2-9b60-43230ae1c48c" />

<img width="908" alt="Image" src="https://github.com/user-attachments/assets/4238d46b-6d6a-44cf-9cb9-b9656a7b1649" />

<img width="945" alt="Image" src="https://github.com/user-attachments/assets/c01e3b62-0dfd-4e47-82d6-03a2dc092a99" />



</Details>

**Comment by tofumatt** [comment]
@kelvinballoo I agree, that's somewhat of an edge case as it's an older version of Safari and a small bug. Given it's working elsewhere it seems like a bug with that browser and not something we can address in that case 😓 

I think it's fine to move to approval in that case 🤔

**Comment by kelvinballoo** [comment]
### QA Update ✅ 

That sounds good @tofumatt .
- Moving this to approval as this was verified good on the devices listed in Details.
- No issues were observed on desktop.

<Details>

Devices - Tested good

<img width="910" alt="Image" src="https://github.com/user-attachments/assets/3e3653bc-5702-49bd-b048-25ed6029d012" />

<img width="1158" alt="Image" src="https://github.com/user-attachments/assets/e82b0362-d34a-4264-851e-51d7a65aa1e7" />

<img width="507" alt="Image" src="https://github.com/user-attachments/assets/54c3752d-2c08-42a2-9b60-43230ae1c48c" />

<img width="908" alt="Image" src="https://github.com/user-attachments/assets/4238d46b-6d6a-44cf-9cb9-b9656a7b1649" />

<img width="945" alt="Image" src="https://github.com/user-attachments/assets/c01e3b62-0dfd-4e47-82d6-03a2dc092a99" />



</Details>
